### ISSUE TITLE
Portuguese terms of service not loading

### ISSUE THREAD

**Issue by dd-pardal** [mozilla/voice-web#2413]
The Portuguese terms of service (https://voice.mozilla.org/pt/terms) won't load. Instead, the website shows just a spinner.

**Comment by rileyjshaw** [comment]
Thank you for logging this!

Behind the scenes, we pull our terms from https://github.com/mozilla/legal-docs/tree/master/Common_Voice_Terms. It looks like we currently don't have a file called `pt.md` – only `pt-BR.md`. My guess is that this is causing the bug.

**Comment by phirework** [comment]
The additional context here is that back in the summer, we decided to generalize Portuguese translations to `pt` instead of `pt-BR`, which we had previously been using. Migrating the terms over to the new value must've slipped through the cracks, so I've submitted a PR to remedy that. 

Here's [more context for this change](https://discourse.mozilla.org/t/concerns-about-brazilian-portuguese-and-dialects-of-other-languages/48767), in case you're curious. Thanks for reporting!
